packages/apps/src/Root.tsx

```diff
@@ -13,6 +13,7 @@ import { ApiStatsCtxRoot, BlockAuthorsCtxRoot, BlockEventsCtxRoot, KeyringCtxRoo
 import { settings } from '@polkadot/ui-settings';
 
 import Apps from './Apps.js';
+import { CustomTheme } from './customTheme.js';
```

Comment:
Can we rename it to QNFTheme.js? if it possible, then let's rename files as well.

Reply:
Fixed